src/components/ui/slider.tsx

```
@@ -1,61 +1,61 @@
+			>
+				<SliderPrimitive.Range
+					data-slot="slider-range"
+					className={cn(
+						"bg-primary absolute data-[orientation=horizontal]:h-full data-[orientation=vertical]:w-full",
+					)}
+				/>
+			</SliderPrimitive.Track>
+			{Array.from({ length: _values.length }, (_, index) => (
+				<SliderPrimitive.Thumb
+					data-slot="slider-thumb"
+					key={index}
```

Biome (2.4.4)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)